api/src/main/java/org/apache/apex/api/EmbeddedAppLauncher.java

```diff
@@ -65,6 +66,41 @@ public static EmbeddedAppLauncher newInstance()
   public interface EmbeddedAppHandle extends AppHandle {}
```

Comment:
Is this a newly introduced interface? Should it be declared as evolving? Please see my comment regarding Exception.